app/models/frontend/expenses/tailoring/ExpensesTailoringCategoryTypeAnswer.scala

```diff
@@ -18,12 +18,12 @@ package models.frontend.expenses.tailoring
 
 import play.api.libs.json.{Json, OFormat, OWrites, Reads}
 
-final case class ExpensesTailoringNoExpensesAnswers(expensesCategories: ExpensesCategories)
+final case class ExpensesTailoringCategoryTypeAnswer(expensesCategories: ExpensesCategories)
```

Comment:
We could probably get away with only having the model `ExpensesTailoringTotalAmountAnswers`, not sure we need this one as well because we'll never have the FE call the BE with only answers to expenses categories, there will always have to be an amount tied to it, right? If so, simplifies things a little.

Reply:
if the answer is NoExpenses then will save only the Category type. I renamed it from 'NoExpensesAnswers' to 'CategoryTypeAnswers' because the method is used both for a 'NoExpenses' and 'TotalAmount' answer so made tried to make it more clear for what it does rather than the answer that is calling it